### PR TITLE
Implement additional buffer opening functions.

### PR DESCRIPTION
**Pull request by marshall-lee**

### CODE REVIEW

**Comment by calleluks** [comment]
Hey @marshall-lee, thanks for contributing! I don't personally use these commands very often but the code sure looks fine to me. @teoljungberg, what are your thoughts?

File: plugin/pick.vim [hunk 1]
```diff
@@ -39,6 +39,18 @@ function! PickBuffer()
   call PickCommand(s:BufferListCommand(), "", ":buffer")
 endfunction
 
+function! PickBufferVerticalSplit()
+  call PickCommand(s:BufferListCommand(), "", ":vert sbuffer")
```
Comment (teoljungberg) [inline plugin/pick.vim:43]: Could this be `:vert buffer` instead?

File: plugin/pick.vim [hunk 2]
```diff
@@ -39,6 +39,18 @@ function! PickBuffer()
...
+endfunction
+
+function! PickBufferSplit()
+  call PickCommand(s:BufferListCommand(), "", ":sbuffer")
+endfunction
+
+function! PickBufferTab()
+  call PickCommand(s:BufferListCommand(), "", ":tab sbuffer")
```
Comment (teoljungberg) [inline plugin/pick.vim:51]: Same as above.

**Comment by teoljungberg** [comment]
I use neither of the commands, but I can see the value in having them.

I left some minor comments / questions on the implementation.

**Comment by calleluks** [comment]
Hey @marshall-lee, are you still interesting in working on this?

**Comment by teoljungberg** [comment]
Hi @marshall-lee, I just merged #12 which adds the funciton `PickBufferCommand` where you can add your own buffer commands.

Since that resolves this issue, I'm closing this PR. Thanks for your contribution!

**Pull request closed without merging**